spec/support/path.rb

```diff
@@ -15,7 +15,9 @@ def home(*path)
     end
 
     def default_bundle_path(*path)
-      system_gem_path(*path)
+      root = tmp.join("bundled_app")
+      FileUtils.mkdir_p(root)
+      root.join(".bundle").join(*path)
```

Comment:
You can definitely implement this in terms of `bundled_app`, defined below. Something like `bundled_app(".bundle", Bundler.ruby_scope)` should do it.